services/web-server/procs.yml

```diff
@@ -3,7 +3,6 @@ web:
   command: node services/web-server/src/main.js server
   readinessPath: .well-known/apollo/server-health
   paths:
-    - '/login'
     - '/login/*'
```

Comment:
this should be safe, as I couldn't find any references for the strict match, all routes are only `/login/something`